examples/terraform/README.md

```diff
@@ -0,0 +1,17 @@
+# Terraform
+This example demonstrates how to use Zarf to execute Terraform code to create an S3 bucket.
+
+### Assumptions/Prereqs
```

Comment:
Just asking do you need the AWS CLI, if so please add that.

Reply:
No AWS CLI necessary, TF interacts with AWS through an [AWS Provider](https://registry.terraform.io/providers/hashicorp/aws/latest/docs)

Reply:
(which is present in the Zarf package)